lib/lotus/model/mapping/coercer.rb

```diff
@@ -58,17 +58,17 @@ def deserialize_#{ mapped }(value)
           instance_eval <<-EVAL, __FILE__, __LINE__
```

Comment:
Non related question: is there a way to use `define_method` here http://tenderlovemaking.com/2013/03/03/dynamic_method_definitions.html? Also, I put in my TODO list to simplify this ;)